src/api/controllers/credentials.ts

```diff
@@ -83,23 +83,45 @@ export class Credentials {
 			type: undefined
 		}
 
-		const credential: CredentialPayload = {
-			issuer: { id: issuer_id.did },
-			'@context': VC_CONTEXT,
-			type: ['Person', VC_TYPE],
-			issuanceDate: new Date().toISOString(),
-			credentialSubject: credential_subject,
-			'WebPage': [
-				{
-					'@type': 'ProfilePage',
-					description: 'Twitter',
-					name: `${user?.nickname}` ?? '<unknown>',
-					identifier: `@${user?.handle}` ?? '<unknown>',
-					URL: `https://twitter.com/${user?.handle}`,
-					lastReviewed: user?.updated_at
+		let credential: CredentialPayload
+		switch(new URL(request.url).searchParams.get('type')) {
+			case 'Ticket':
+				credential = {
+					issuer: { id: issuer_id.did },
+					'@context': VC_CONTEXT.concat(VC_EVENTRESERVATION_CONTEXT),
+					type: ['EventReservation', VC_TYPE],
+					issuanceDate: new Date().toISOString(),
+					credentialSubject: credential_subject,
+					reservationId: new URL(request.url).searchParams.get('data'),
+					reservationStatus: 'https://schema.org/ReservationConfirmed',
+					reservationFor: {
+						'@type': 'Event',
+						name: 'IIW Event',
+						startDate: "2022-11-06T19:30:00-08:00",
```

Comment:
Check ClickUp for ticket body

Reply:
@ankurdotb updated this, can you have a look